docs(iam): add user mapping to identity federation (#6755)
scaleway/docs-content: pages/iam/how-to/set-up-identity-federation.mdx

<Message type="important">
  Keep in mind that:

>   - User mapping between Scaleway and the Identity Provider must be done on the username. When testing the connection, if an error persists about the user not existing in Scaleway, make sure your Identity Provider is mapping the user on their username

Reviewer: ```suggestion
  - User mapping between Scaleway and the Identity Provider must be done on the username. When testing the connection, if an error persists about the user not existing in Scaleway, make sure your Identity Provider is mapping the user on their username.
```

  - Members need to already have been [created manually](/iam/how-to/manage-members/#how-to-create-a-member) in Scaleway to log in.
  - If you delete a user in the IdP, the corresponding Member is not automatically deleted in your Scaleway Organization. The [deletion must happen manually](/iam/how-to/manage-members/#how-to-delete-a-member).
</Message>

## Map user attributes

Scaleway matches Members using their username, received as the SAML `NameID` value, not their email address. Incorrect attribute mapping in the Identity Provider is a common cause of connection failures, where the Member cannot be found in Scaleway.

<Message type="important">

>   The value sent as the SAML `NameID` must correspond to the username of the Member in Scaleway, not their email address, unless the two values happen to be identical. If a connection test fails with an error about the user not existing in Scaleway, check the attribute mapping in the identity provider first.

Reviewer: ```suggestion
  The value sent as the SAML `NameID` must correspond to the username of the Member in Scaleway, not their email address, unless the two values happen to be identical. If a connection test fails with an error about the user not existing in Scaleway, check the attribute mapping in the Identity Provider first.
```

</Message>

### Google Workspace

> On the **Attribute mapping** page of the Scaleway SAML app in Google Workspace, no additional mapping is generally required for SSO to work. The essential mapping is the following:

Reviewer: ```suggestion
On the **Attribute mapping** page of the Scaleway SAML app in Google Workspace, no additional mapping is generally required for SSO to work. Verify that the following attributes are mapped as follows:
```

Reviewer: I am suggesting the change because as a user, I was wondering what "essential mapping" means, whether I have to do anything. 
I don't know the context so the suggestion may not be fully correct. Feel free to modify or ignore.

> On the **Attribute mapping** page of the Scaleway SAML app in Google Workspace, no additional mapping is generally required for SSO to work. Verify that the following attributes are mapped as follows:

Reviewer 2: Slight repetition following + follows. I know English is a lot more permissive about those, but I like to avoid them x) Also, based on past conversations, shouldn't we use "check" rather than "verify" here?

"Check that the following attributes are mapped as indicated" ou "as specified" ?
